content/en/security_platform/cspm/getting_started.md

```diff
@@ -19,30 +19,200 @@ Cloud Security Posture Management is not currently available in this site.
 </div>
 {{< /site-region >}}
 
-## Overview
+Cloud Security Posture Management (CSPM) makes it easier to assess and visualize the current and historic security posture of your cloud resources, automate audit evidence collection, and catch misconfigurations that leave your organization vulnerable to attacks.
 
-Navigate to the [Getting Started page][1] to configure your environment for scanning.
+## Enable CSPM for your cloud resources
 
-{{< img src="security_platform/cspm/getting_started/posture-management-setup.png" alt="Setup page for CSPM" style="width:100%;">}}
+CSPM provides agentless onboarding using existing Datadog integrations with cloud providers such as AWS, Azure, GCP, Docker, and Kubernetes. For details on how to configure CSPM, select your cloud provider and follow the instructions:
 
-## Setup
+{{< tabs >}}
+{{% tab "AWS" %}}
 
-### Product configuration
+### Set up the Datadog AWS integration
 
-Select `Posture Management`. You can also configure multiple products at once by selecting their checkbox.
+If you haven't already, set up the [Amazon Web Services integration][1]. For CSPM, you must also add the [necessary permissions][2] for resource collection.
 
-### Assess your cloud environment
+### Enable CSPM for AWS
 
-Detect misconfigurations in your cloud environment. This section guides you through collecting resource configuration data from cloud providers. Resource configuration collection allows Datadog to assess your environments against Datadog’s out-of-the-box [Posture Management cloud configuration detection rules][2].
+Use one of the following methods to enable CSPM for your AWS accounts:
 
-### Assess your host and containers
+#### Security Setup & Configuration
 
-Evaluate the security posture of your hosts and containers. This section guides you through configuring the Datadog Agent to scan your hosts and containers. Our agent allows Datadog to continuously assess the state of your hosts and containers against Datadog’s out-of-the-box [Posture Management Infrastructure Configuration detection rules][3].
+1. Navigate to **Security** > **Setup and Configuration**.
+2. Follow the [in-app instructions][3] to activate CSPM for your account.
```

Comment:
for every "in-app instructions", would be great to link to the specific cloud provider

for AWS: https://app.datadoghq.com/security/configuration?sectionId=secureCloudEnvironment&secure-cloud-environment=amazon-web-services

for Azure: https://app.datadoghq.com/security/configuration?sectionId=secureCloudEnvironment&secure-cloud-environment=azure

for GCP: https://app.datadoghq.com/security/configuration?sectionId=secureCloudEnvironment&secure-cloud-environment=google-cloud-platform

Reply:
My idea was that this step would be for customers who are activating CSPM for the first time. Let me know if that's not clear with the way I described it, and I'll modify the verbiage.

On that note, I think it's a good idea. I'll incorporate your suggestion by linking directly to each cloud provider in Step 3.